_delb/nodes.py

```diff
@@ -2782,6 +2782,12 @@ class QueryResults(Sequence[TagNode]):
     def __init__(self, results: Iterator[_ElementWrappingNode]):
         self.__items = cast(Tuple[TagNode], tuple(results))
 
+    def __eq__(self, other):
+        if not isinstance(other, QueryResults):
```

Comment:
not sure whether all Sequences should be supported.

Reply:
why not?

Reply:
it'd require more tests… i'm not in the mood.

Reply:
and i wonder what significance order has.

Reply:
ok good point

Reply:
i unreoslved this b/c i wanna give it some thought.

Reply:
luckily [the words](https://www.w3.org/TR/1999/REC-xpath-19991116/#section-Introduction) are very clear right at the beginning:

> node-set (an unordered collection of nodes without duplicates)

i haven't read until the end whether they came up with the idea to coerce that into another type.

hence it should be compared like a set. but what if the other is a list that contains a node twice?